site/components/site-header.tsx

```diff
@@ -0,0 +1,122 @@
+import { PropsWithChildren, ReactElement } from 'react'
+import Link from 'next/link'
+import Image from 'next/image'
+import { animatedUnderline, colors, mediaQueries } from '../utils'
+
+type HeaderLinkProps = PropsWithChildren<{ href: string; active?: boolean }>
+
+function HeaderLink({
+  href,
+  active = false,
+  children
+}: HeaderLinkProps): ReactElement {
+  // TODO:SAM active styles
+  return (
+    <Link href={href}>
+      <a
+        css={[
+          {
+            fontWeight: 500
+          },
+          animatedUnderline
+        ]}
+      >
+        {children}
+      </a>
+    </Link>
+  )
+}
+
+export function SiteHeader() {
+  return (
+    <header
+      css={{
+        gridColumn: '1 /span 2',
+        display: 'flex',
+        alignItems: 'center'
+      }}
+    >
+      <Link href="/">
+        <a
+          css={{
+            display: 'flex',
+            alignItems: 'center',
+            color: colors.pink,
+            textDecoration: 'none',
+            marginRight: '1rem'
+          }}
+        >
+          <Image alt="Avatar" src="logo.png" height={36} width={36} />
+          <h4
+            css={{
+              display: 'none',
+              margin: 0,
+              marginLeft: '1rem',
+              fontWeight: 'bold',
+              [mediaQueries.lgUp]: {
+                display: 'inline'
+              }
+            }}
+          >
+            Emotion
+          </h4>
+        </a>
+      </Link>
+      <nav
+        css={{
+          marginLeft: 'auto',
+          overflowX: 'auto'
+        }}
+      >
+        <ul
+          css={{
+            display: 'grid',
+            gridAutoFlow: ['column dense', undefined, 'column'],
+            gridColumn: ['2 / span 1', undefined, 'auto'],
+            gridRow: ['1', '1', 'auto'],
```

Comment:
If I'm not mistaken - this is not what you are looking for. This won't use those values for different media breakpoints, for it to work like that you have to use `facepaint`.